ci/test.yml

```diff
@@ -308,19 +308,21 @@ connect-popup legacy npm package:
     refs:
       - schedules
 
-.e2e connect-popup-webextension:
+.e2e connect-explorer-webextension:
   stage: integration testing
-  retry: 1
+  retry: 0
```

Comment:
nice, lets now rebase this couple of times and check https://gitlab.com/satoshilabs/trezor/trezor-suite/-/pipelines?page=1&scope=all&ref=feat%2Ftrezor-connect-proxy-with-popup-tests-webextension 
whther this pipeline is green or not. with retry on, some problem might stay hidden